Line/server/config.json

```diff
@@ -0,0 +1,5 @@
+{
+  "line": {
+    "channelId": "<your_channel_id>"
+  }
+}
```

Comment:
Fixed in this [commit](https://github.com/firebase/custom-auth-samples/pull/1/commits/7133cb86f44d6d2175b8059482815ba537cf5f64) :)